docs/development.md

```diff
@@ -1,6 +1,10 @@
 # Développer sur Inclusion Connect
 
-NB: Pour l'instant la procédure qui suit ne fonctionne pas. À corriger lors de la mise à jour de KeyCloak
+NB(alaurent): Avec le dernier docker existant, j'arrive à lancer keycloak en local.
+Par contre, à chaque fois qu'on veut modifier le theme, il faut re-créer l'image docker, et donc
+supprimer l'existante avec `docker rmi itou-inclusion-connect_keycloak`
+Ce qui fait qu'on perd aussi toute la configuration réalisée (création du realm, du client pour s'y connecter, choix du theme, etc)
+Il faudrait donc créer un script pour le faire automatiquement (utiliser un makefile plutot que docker compose ?)
```

Comment:
super commentaire, à bien mettre en avant pour que les gens voient que c'est un work in progress.